graphql/schema.graphql

```diff
@@ -2,7 +2,7 @@ input StoreDashInput {
   appName: String!
   appVersion: String
   region: String
-  production: Boolean
+  production: String
```

Comment:
nitpicking: why not boolean?

Reply:
Because this param is passed to storedash as a string of `true` or `false`. So to avoid conversions, I made it a `string` in both front-end and back-end.